app/assets/stylesheets/mixins.less

```diff
@@ -9,18 +7,27 @@
     user-select: none;
 }
 
-.vertical-scroll-disabled {
```

Comment:
This was taken out, but you didn't implement an alternate way of stopping the conversation page from scrolling when the popup is open. Is this intended?